projects/proposals/z:OSMF Workflows.md

```diff
@@ -0,0 +1,60 @@
+* Name of project (must be unique within OMP)
+z/OSMF Workflows
+* Project description (what it does, why it is valuable, origin and history)
+
+This projects enables customers to share z/OSMF workflows. z/OSMF workflows modernizes the system management tasks performed day to day by z/OS System Programmers. Sharing of z/OSMF workflows for system management will help customers to learn from each other.
+
+* Statement on alignment with OMP charter mission
+
+OMP is for mainframe cooperation and it's helpful to fulfill our mission to collaborate on z/OSMF workflows development.
+
+* Sponsor from TSC (sponsor helps mentor projects)
+
+John Mertic
```

Comment:
You should have a TSC member here - I'm not one of them ;-)

See the list at https://github.com/openmainframeproject/tsc#voting-tsc-members

Reply:
This is a new project specific to z/OSMF. I don't know who from TSC member I can pick. Best I can think of is Matt Hogstrom.

Reply:
OK, adding @hogstrom to the review for his input.

Reply:
@shreeji818 Can you put Matt Hogstrom down as the TSC sponsor?